repos/spack_repo/builtin/packages/amdlibm/package.py

```diff
@@ -47,14 +49,17 @@ class Amdlibm(SConsPackage):
 
     variant("verbose", default=False, description="Building with verbosity", when="@:4.1")
 
+    # Build system
+    build_system(conditional("cmake", when="@5.1"), "scons", default="scons")
```

Comment:
Unless you'll remove this for the next version:
```suggestion
    build_system(conditional("cmake", when="@5.1:"), "scons", default="scons")
```

Reply:
Updated as suggested